api/server.mjs

```diff
@@ -17,6 +18,7 @@ server
   .use(cors())
   // Enable gzip
   .use(compress())
+  .use(koaStatic('.'))
```

Comment:
What are these API changes for?

Reply:
For access Uploaded images from Mobile app.